modules/awesomebooks.js

```diff
@@ -0,0 +1,39 @@
+import Book from './book.js';
+
+const booksArray = JSON.parse(localStorage.getItem('books')) || [];
+const listSec = document.getElementById('book-list');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+
+export default class AwesomeBooks {
+  static showList() {
+    let listHtml = '';
+    for (let i = 0; i < booksArray.length; i += 1) {
+      listHtml += `
+        <div class="book-row even-odd-${i % 2}" id="book-${i}">
+          <p>"${booksArray[i].title}"   by   ${booksArray[i].author}</p>
+          <button id="btn-book-${i}" data-index=${i} class="btn-remove">Remove</button> 
+        </div>`;
+    }
+    listSec.innerHTML = listHtml;
+  }
+
+  static addBook(e) {
+    if (title.value.length > 0 && author.value.length > 0) {
+      e.preventDefault();
+      booksArray.push(new Book(title.value.toUpperCase(), author.value.toUpperCase()));
+      title.value = '';
+      author.value = '';
+      AwesomeBooks.showList();
+      localStorage.setItem('books', JSON.stringify(booksArray));
+    }
+  }
```

Comment:
-  [ ] OPTIONAL
  * Books are stored correctly in the local storage when a book detail is entered, however when the page is refreshed the book remains in the local storage but however vanishes from the list of books page. Please ensure that data is both correctly displayed and reflected in the local storage even when the page is reloaded.

  * Also, it's not required to change data to uppercase before adding it to the local storage

**Missing book list:**
![image](https://user-images.githubusercontent.com/12745474/213086746-2da65ad8-0769-44b5-864b-bc6847a570bb.png)